apps/web/package.json

```diff
@@ -180,6 +182,7 @@
     "deasync": "^0.1.30",
     "detect-port": "^1.3.0",
     "env-cmd": "^10.1.0",
+    "google-auth-library": "^9.15.0",
```

Comment:
Adding this as a dev dependency because we need to import type `Credentials` from the lib.